### PR TITLE
[Maintainers pilot] feat: Add catalog-info.yaml

### PR DESCRIPTION
**Pull request by giovannicimolin**
@schenedx I've been invited to be a part of the [maintainer's program](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html) to help take care of this repository ([reference](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3426844690/Bootstrapping+Maintainership)). 

Since your team was originally responsible for this repository (IIRC, correct if I'm wrong), let me know if you want to keep this role internal. I don't want to step in front of anyone on this, so I'm pinging just in case you weren't aware.

----

This PR adds the `catalog-info.yaml` file so that backstage can extract metadata from this repository.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @giovannicimolin!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by giovannicimolin** [comment]
@Zacharis278 @MichaelRoytman Do you have a GitHub handle for your team? If not, can you set one up?

I only found @openedx/masters-devs-cosmonauts, is this the right one?

Once this is set up I'll update the PR and add the repo to Backstage as per https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.html.

**Comment by MichaelRoytman** [comment]
@giovannicimolin Yes, this is the correct handle.

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      title: Grimoire Pull Request Backlog
+      icon: dashboard
+spec:
+  type: XBlock
+  lifecycle: production
+  owner: user:giovannicimolin
```
Comment (Zacharis278) [inline catalog-info.yaml:15]:
```suggestion
  owner: masters-devs-cosmonauts

```

Comment (Zacharis278) [reply catalog-info.yaml:15]: @giovannicimolin think this is all that needed to change on this

**Comment by Zacharis278** [comment]
@giovannicimolin I opened another PR to handle the README updates so I just copied this change there https://github.com/openedx/xblock-lti-consumer/pull/282

**Comment by giovannicimolin** [comment]
@Zacharis278 I missed your pings on this, sorry! 
I'll review #282 and close this one.

**Comment by openedx-webhooks** [comment]
@giovannicimolin Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/xblock-lti-consumer&entry.867055334=https://github.com/openedx/xblock-lti-consumer/pull/274&entry.1484655318=https://github.com/giovannicimolin&entry.752974735=2022-08-15+19:20&entry.1917517419=2022-09-13+19:40&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**